INSTALL.md

```diff
@@ -0,0 +1,87 @@
+###### Building Endless Sky
+_NOTE: If you have strayed here by accident and are inexperienced with building
+software and the command line, it may be a better idea to grab the program from
+your package manager, the releases section in GitHub, or Steam, rather than
+fruitlessly wasting your time_
```

Comment:
This file seems to be somewhat redundant with `readme-developer.txt`. Perhaps one should be merged into the other, or the data otherwise deduplicated and a reference to the other file added.